### PR TITLE
Fix build and tests on NetBeans 8.2

### LINKED ISSUE #103: Netbeans 8.2
When installing this plugin (v0.10.1)  I get the following exception when attempting to open any projects :

v0.10.0 Works as expected

Product Version: NetBeans IDE 8.2 (Build 201609300101)
Java: 1.8.0_101; Java HotSpot(TM) 64-Bit Server VM 25.101-b13
Runtime: Java(TM) SE Runtime Environment 1.8.0_101-b13
System: Linux version 4.4.0-47-generic running on amd64; UTF-8; en_GB (nb)
Plugin Version : v0.10.1

    java.lang.ClassNotFoundException: org.netbeans.spi.project.ui.ProjectOpenedHook
    	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
    	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
    	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    	at org.netbeans.ProxyClassLoader.loadClass(ProxyClassLoader.java:222)
    Caused: java.lang.ClassNotFoundException: org.netbeans.spi.project.ui.ProjectOpenedHook starting from ModuleCL@5260eb1e[com.welovecoding.netbeans.plugin.editorconfig] with possible defining loaders [ModuleCL@6dc4425e[org.netbeans.modules.projectuiapi], ModuleCL@702dbb46[org.netbeans.modules.projectuiapi.base]] and declared parents [ModuleCL@61e7df45[org.netbeans.modules.parsing.indexing], ModuleCL@6c644b11[org.netbeans.modules.queries], org.netbeans.MainImpl$BootClassLoader@7daf6ecc, ModuleCL@798b2cba[org.netbeans.modules.csl.api], ModuleCL@3d835f75[org.openide.filesystems.nb], ModuleCL@2007f2f7[org.netbeans.modules.projectapi], ModuleCL@124e8334[org.netbeans.modules.editor.mimelookup], ModuleCL@5d036e38[org.netbeans.core.multiview], ModuleCL@3c594181[org.openide.loaders], ModuleCL@5f4b1991[org.netbeans.modules.csl.types], ...17 more]
    	at org.netbeans.ProxyClassLoader.loadClass(ProxyClassLoader.java:224)
    	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    Caused: java.lang.NoClassDefFoundError: org/netbeans/spi/project/ui/ProjectOpenedHook
    	at java.lang.ClassLoader.defineClass1(Native Method)
    	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
    	at org.netbeans.JarClassLoader.doLoadClass(JarClassLoader.java:311)
    	at org.netbeans.ProxyClassLoader.selfLoadClass(ProxyClassLoader.java:259)
    Caused: java.lang.NoClassDefFoundError: org/netbeans/spi/project/ui/ProjectOpenedHook while loading com.welovecoding.nbeditorconfig.listener.ProjectOpenCloseListener; see http://wiki.netbeans.org/DevFaqTroubleshootClassNotFound
    	at org.netbeans.ProxyClassLoader.selfLoadClass(ProxyClassLoader.java:263)
    	at org.netbeans.ProxyClassLoader.loadClass(ProxyClassLoader.java:187)
    	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    	at com.welovecoding.nbeditorconfig.listener.ProjectHookLookup.createAdditionalLookup(ProjectHookLookup.java:53)
    	at org.netbeans.spi.project.support.DelegatingLookupImpl.doDelegate(DelegatingLookupImpl.java:139)
    	at org.netbeans.spi.project.support.DelegatingLookupImpl.<init>(DelegatingLookupImpl.java:91)
    	at org.netbeans.spi.project.support.LookupProviderSupport.createCompositeLookup(LookupProviderSupport.java:95)
    	at org.netbeans.modules.php.project.PhpProject.createLookup(PhpProject.java:781)
    	at org.netbeans.modules.php.project.PhpProject.<init>(PhpProject.java:306)
    Caused: java.lang.reflect.InvocationTargetException
    	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
    	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
    	at org.netbeans.modules.project.ant.AntBasedGenericType.createProject(AntBasedGenericType.java:131)
    Caused: java.lang.IllegalArgumentException
    	at org.netbeans.modules.project.ant.AntBasedGenericType.createProject(AntBasedGenericType.java:140)
    	at org.netbeans.modules.project.ant.AntBasedProjectFactorySingleton.loadProject(AntBasedProjectFactorySingleton.java:264)
    	at org.netbeans.modules.projectapi.nb.NbProjectManager.createProject(NbProjectManager.java:401)
    	at org.netbeans.modules.projectapi.nb.NbProjectManager.access$300(NbProjectManager.java:94)
    	at org.netbeans.modules.projectapi.nb.NbProjectManager$2.run(NbProjectManager.java:314)
    	at org.netbeans.modules.projectapi.nb.NbProjectManager$2.run(NbProjectManager.java:243)
    	at org.netbeans.modules.openide.util.DefaultMutexImplementation.readAccess(DefaultMutexImplementation.java:209)
    	at org.openide.util.Mutex.readAccess(Mutex.java:250)
    	at org.netbeans.modules.projectapi.nb.NbProjectManager.findProject(NbProjectManager.java:243)
    	at org.netbeans.api.project.ProjectManager.findProject(ProjectManager.java:167)
    	at org.netbeans.modules.welcome.content.RecentProjectsPanel$OpenProjectAction$1.run(RecentProjectsPanel.java:232)
    	at org.netbeans.modules.progress.ui.RunOffEDTImpl$1.run(RunOffEDTImpl.java:169)
    	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1443)
    	at org.netbeans.modules.openide.util.GlobalLookup.execute(GlobalLookup.java:68)
    	at org.openide.util.lookup.Lookups.executeWith(Lookups.java:303)
    [catch] at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:2058)

### PR DESCRIPTION
**Pull request by Jenselme**
Close #103

### CODE REVIEW

File: src/main/java/com/welovecoding/nbeditorconfig/processor/EditorConfigProcessor.java [hunk 1]
```diff
@@ -43,7 +43,8 @@
   public EditorConfigProcessor() {
     // Project nbProject = OpenProjects.getDefault().getMainProject();
     Lookup lookup = Utilities.actionsGlobalContext();
-    Project project = lookup.lookup(Project.class);
+//    Project project = lookup.lookup(Project.class);
```
Comment (bennyn) [inline src/main/java/com/welovecoding/nbeditorconfig/processor/EditorConfigProcessor.java]: Commented code can be removed

**Comment by bennyn** [comment]
Hi @Jenselme. Thank you very much for your contribution! Just remove the commented code and then it can be merged. 😎

File: src/main/java/com/welovecoding/nbeditorconfig/processor/EditorConfigProcessor.java (hunk 1 above)
Comment (Jenselme) [reply src/main/java/com/welovecoding/nbeditorconfig/processor/EditorConfigProcessor.java]: I think there is more to that. If you look at the code, I initialize project to `null` then only test it to see if it's `null`. Quite useless. I cannot remember why I did this, it's a bit old now. Perhaps this is why I didn't create the PR. I'll have to complete that tonight.

**Comment by Jenselme** [comment]
@bennyn just remove useless code. I don't know why I changed `EditorConfigProcessor.java`. It seems to work fine as it is. Some extra tests could be good thought. I tested today on a different 8.2 installation. Maybe my problem came from a strange interaction with another plugin.

**Comment by Chris2011** [comment]
Great, thx. After the merge I will contribute too.

**Comment by bennyn** [comment]
@Jenselme: Thanks you very much for your contribution!! 🍻

@Chris2011: I'll take you to the word!

**Pull request merged**